packages/esm-patient-chart-app/src/clinical-views/encounter-tile/tile.scss

```
@@ -18,17 +18,11 @@
   @include type.type-style('body-long-01');
   display: block;
   padding-top: layout.$spacing-05;
-}
-
-.columnContainer {
-  display: flex;
-  flex-direction: row;
-  gap: layout.$spacing-05;
-  justify-content: space-around;
-}
-
-.tileSubTitle {
-  @include type.type-style('body-compact-01');
+  font-size: var(--cds-label-01-font-size, 0.75rem);
+  line-height: var(--cds-label-01-line-height, 1.33333);
+  letter-spacing: var(--cds-label-01-letter-spacing, 0.32px);
```

I think this is equivalent right?

```suggestion
  @include type.$label-01;
```

---

Yep! Minus the font color